src/http-spec.ts

```diff
@@ -236,3 +237,7 @@ export interface IOauth2PasswordFlow extends IOauth2Flow {
 export interface IOauth2ClientCredentialsFlow extends IOauth2Flow {
   tokenUrl: string;
 }
+
+export interface Extensions {
+  [key: string]: any;
+}
```

Comment:
We could have it as 
```ts
export interface Extensions {
  [key: `x-${string}`]: any;
}
```
but then we'd need to update typescript here and in all packages using it